### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by LaurenQurashi**
## Description of change

This PR updates the dependencies from Dependabot.

It also includes updating a dependency for browserlist for the visual tests, as well as adding in a babel typescript preset in order to avoid some errors when running storybook.  

## Test instructions

_What should I see?_

### Before

_Add a screenshot_

### After

_Add a screenshot_

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

**Review by PippoRaimondi — APPROVED** [review]
Leaving this pre-approved pending the tests pass!

File: package.json [hunk 1]
```diff
@@ -171,8 +171,12 @@
     "winston": "^3.3.3"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "7.16.0",
```
Comment (craigjbass) [inline package.json:174]: This dependency is no longer needed, as it's bundled in @babel/preset-env
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -171,8 +171,12 @@
...
+    "@babel/plugin-proposal-object-rest-spread": "7.16.0",
```
Comment (craigjbass) [inline package.json:175]: This dependency is no longer needed, as it's bundled in @babel/preset-env
```suggestion
```

**Review by craigjbass — APPROVED** [review]

**Comment by craigjbass** [comment]
Because I know how painful this PR is we can remove those plugin dependencies (again) in a followup PR

**Pull request merged**